### DISCUSSION TITLE
Problème génération course vigne

### DISCUSSION THREAD

**Discussion by FK2316** [Courseplay/Courseplay_FS22#3095] [Q&A]
![20240207132057_1](https://github.com/Courseplay/Courseplay_FS22/assets/159262293/5fe69672-fb42-4189-af3e-85002c1d8c83)

Bonjour,
Impossible de générer une autre course que celle présenté pour les vignes. Comment faire ? merci

**Comment by Tensuko** [comment]
https://github.com/Courseplay/Courseplay_FS22/issues/1683
